components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java

```diff
@@ -627,6 +638,15 @@ private boolean doThrottle(MessageContext messageContext) {
             initThrottleForSubscriptionLevelSpikeArrest(messageContext, authenticationContext);
             context.stop();
         }
+
+        if(authenticationContext != null && authenticationContext.getApplicationSpikesArrestLimit() > 0) {
```

Comment:
format

Reply:
fixed in b56bf3020abe507022f90c1b46e64547b10cf7ac